action/run-in-renode.py

```diff
@@ -511,20 +529,6 @@ def setup_renode():
         run_cmd(child, "#", "ip route add default via 172.16.0.1")
         run_cmd(child, "#", 'echo "nameserver 1.1.1.1" >> /etc/resolv.conf')  # adds dns server address
 
-        now = datetime.now()
-
-        run_cmd(child, "#", f'date -s "{now.strftime("%Y-%m-%d %H:%M:%S")}"')
-
-        # increase git buffers
-        # mitigates issues with broken pipe `Send failure: Broken pipe`
-        run_cmd(child, "#", 'git config --global http.maxRequestBuffer 240M')
-        run_cmd(child, "#", 'git config --global http.postBuffer 100M')
-        run_cmd(child, "#", 'git config --global core.compression 0')
-
-        run_cmd(child, "#", 'git config --global --unset https.proxy')
-        run_cmd(child, "#", 'git config --global --unset http.proxy')
-        run_cmd(child, "#", 'git config --global ssl.Verify false')
```

Comment:
Why was this removed?

Reply:
I think this is not required. I added this because I think it will fixed the issues with https connections, but I do not think that this mitigations even does something.